src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      count++;
+
+      if (count > 3 && (count % 2 === 0)) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstNum + secondNum;
```

Comment:
basically  it should will work without else

Reply:
Good comment from your colleague. It will work without `else`

Reply:
Okay, got it)